ztp/source-crs/PtpConfigBoundaryForEvent.yaml

```diff
@@ -16,13 +16,13 @@ spec:
       logReduce: "true"
     ptp4lConf: |
       # The interface name is hardware-specific
-      [$iface_slave]
+      [(?<iface_slave>[[:alnum:]]+)]
```

Comment:
@aneeshkp / @jzding / @josephdrichard : Does this look okay?  This changes the example from `$interface_slave` to an actual regex capturegroup `(?<interface_slave>...)` which maintains the idea of "Customers need to fill in this part", but adds the actual regex that the text should match.

It's a side-effect of the new facility we added in the reference, and if we really need to fixup the source-crs so they don't change, we can do that...